src/api/app/models/kiwi/image.rb

```diff
@@ -62,6 +63,27 @@ def self.build_from_xml(xml_string, md5)
       new_image.repositories.build(attributes)
       order += 1
     end
+    package_groups = xml["packages"]
+    package_groups = [xml["packages"]] if xml["packages"].is_a?(Hash)
+    package_groups.each do |package_group_xml|
+      attributes = {
+        kind:         package_group_xml['type'],
```

Comment:
Agree with @bgeuken to use kiwi_type to avoid the mess with disable inheritance_column...